auditbeat/docker-compose.yml

```diff
@@ -14,6 +14,7 @@ services:
       - KIBANA_PORT=5601
     volumes:
       - ${PWD}/..:/go/src/github.com/elastic/beats/
+      - /sys/kernel/tracing/:/sys/kernel/tracing/
```

Comment:
don't we need the sys/kernel/debug as well?

Reply:
tracefs should be enough for recent kernels

Reply:
ok cool as long as you are sure that thid is the case for all 5.12+ kernels in all distros we need to support

Reply:
5.10+*

isn't this docker-compose only used in tests? I only added this because tests were failing locally, but I don' think this is supposed to be used in production